config/initializers/rollbar.rb

```diff
@@ -0,0 +1,5 @@
+require 'rollbar'
+
+Rollbar.configure do |config|
+  config.access_token = 'b47585744d104ac283829af936fdc67e'
```

Comment:
do it through env var and add to docker-compose

Reply:
+